src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random randomNumber = new Random();
+        int value = randomNumber.nextInt(100);
+        ColorSupplier supplier = new ColorSupplier();
```

Comment:
make global Random and ColorSupplier fields and use them here, reason is the same as about Random above